src/cljs/main/broadfcui/page/workspace/summary/catalog.cljs

```diff
@@ -91,128 +94,110 @@
-             (if (= typeahead "ontology")
-               (do
-                 (.typeahead (js/$ (@refs property))
-                             (clj->js {:highlight true
-                                       :hint true
-                                       :minLength 3})
-                             (clj->js
-                               {:source options
-                                :display (fn [result]
-                                           (aget result "label"))
-                                :templates (clj->js
-                                             {:empty "<div> unable to find any matches to the current query </div>"
-                                              :suggestion
-                                              (fn [result]
-                                                (str "<div> <div style='line-height: 1.5em;'>" (aget result "label")
-                                                     "<small style='float: right;'>" (aget result "id") "</small></div>"
-                                                     "<small style='font-style: italic;'> " (aget result "definition") "</small></div>"))})}))
-                 (.bind (js/$ (@refs property))
-                        "typeahead:select"
-                        (fn [ev suggestion]
-                          (swap! state update :attributes assoc
-                                 property-kwd (aget suggestion "label")
-                                 (keyword relatedLabel) (aget suggestion "label")
-                                 (keyword relatedID) (aget suggestion "id"))))))))))})
+         (fn [property]
+           (let [current-value (get attributes property)
+                 {:keys [type enum consentCode renderHint] :as prop} (get-in library-schema [:properties property])
+                 {:keys [wording datatype emptyChoice]} renderHint
+                 required? (contains? required-attributes property)
+                 error? (contains? invalid-properties property)
+                 colorize #(merge % (when error? {:borderColor (:exception-state style/colors)
+                                                  :color (:exception-state style/colors)}))
+                 update-property #(swap! state update :attributes assoc property (.. % -target -value))
+                 checkbox (fn [{:keys [val label]}]
+                            [:label {:style (colorize {:display "inline-flex" :alignItems "center"
+                                                       :cursor "pointer" :marginRight "2em"})}
+                             [:input (merge
+                                      {:type "radio" :style {:cursor "pointer"}
+                                       :onClick #(swap! state update :attributes assoc property val)}
+                                      (when (= val current-value) {:checked true}))]
+                             [:div {:style {:padding "0 0.4em" :fontWeight "500"}} (or label (str val))]])]
```

Comment:
should we move this checkbox definition to the common components and call it radio button?
the checkbox that is currently in common components is not generally useful (specific to a read-only checkbox i think)